tree/readspeed/src/ReadSpeedCLI.cxx

```diff
@@ -129,7 +129,18 @@ const auto fullUsageText =
    " first read. If this is realistic of how your analysis will run - then there is no concern. However, if\n"
    " you expect to only read files once in a while - and as such the files are unlikely to be in the cache -\n"
    " consider clearing the cache before running rootreadspeed.\n"
-   "  (On Linux this can be done by running 'echo 3 > /proc/sys/vm/drop_caches' as a superuser)\n";
+   " On Linux this can be done by running 'echo 3 > /proc/sys/vm/drop_caches' as a superuser\n"
+   " or a specific file can be dropped from the cache with"
+   " `dd of=<FILENAME> oflag=nocache conv=notrunc,fdatasync count=0 > /dev/null 2>&1`.\n"
+   "\n"
+   "\n"
+   "Known overhead of TTreeReader, RDataFrame:\n"
+   "`rootreadspeed` is designed to read all data present in the specified branches, trees and files at the highest\n"
+   "possible speed. When an analysis application is bound by I/O, higher-level interfaces built on top of TTree\n"
+   "such as TTreeReader and RDataFrame are known to add a significant runtime overhead with respect to the runtimes\n"
+   "reported by `rootreadspeed` (up to a factor 2). In realistic analysis applications it has been observed that a\n"
```

Comment:
Can you add that this is planned to be addressed by `RDataFrame`'s usage of `RNTuple`?